### PR TITLE
Fixing htop

### PR DESCRIPTION
**Pull request by 5c077m4n**
#13

### CODE REVIEW

**Comment by roee-shapira-namogoo** [comment]
#13

File: src/utils/logging.rs
Comment (imsnif) [inline src/utils/logging.rs:10]: I think if these functions are public we can remove all the leading underscores and not get warnings, right?

**Review by imsnif — CHANGES_REQUESTED** [review]
@5c077m4n and I discussed this offline, so to sum it up here:
While this PR doesn't fully fix htop rendering inside a pane, it's a large step in that direction. We're going to go forward with it and continue fixing what's left in additional PRs.

It also includes some very appreciated house-cleaning!

@5c077m4n - I've left just one minor nitpick in the code. After we get that done, I'll merge this. Thanks! Looking forward to the next PRs. :)

File: src/utils/consts.rs
Comment (henil) [inline src/utils/consts.rs:3]: @5c077m4n A minor nitpick, but can the `mosaic-log` directory and `mosaic-log.txt` be moved inside `mosaic` folder?

Comment (5c077m4n) [reply src/utils/consts.rs:3]: You are right, I just left it as it was so custom scripts won't break @henil

**Comment by imsnif** [comment]
Btw, I'm waiting for you to let me know when you're finished so I can merge :)

**Comment by 5c077m4n** [comment]
@imsnif Sorry 😬 yep, it's done, all tests are passing

**Comment by imsnif** [comment]
@5c077m4n - no worries :) There's still some underscores in front of the logging function names though? Do you think it's an issue to remove them?

**Comment by 5c077m4n** [comment]
@imsnif Done!

**Review by imsnif — APPROVED** [review]
Very cool! Thanks for all the changes

**Pull request merged**
